bin/taskcluster-ssh.js

```diff
@@ -0,0 +1,73 @@
+var taskcluster = require('taskcluster-client');
+var Promise = require('promise');
+var DockerClient = require('docker-exec-websocket-server').DockerExecClient;
+var https = require('https');
+var url = require('url');
+var _ = require('lodash');
+var assert = require('assert');
+require('../lib/config').load();
+
+var INTERACTIVE_SOCKET_NAME = 'private/docker-worker/interactive.sock';
+
+var yargs = require('yargs')
+  .usage('SSH into task container')
+  .example('taskcluster ssh', 'taskId')
+
+var taskId = yargs.argv._[0];
+
+var queue = new taskcluster.Queue();
+
+queue.status(taskId).then(function(result) {
+  // Get run and check state
+  var run = _.last(result.status.runs);
+  console.log("Latest run: " + run.runId);
+  if (run.state !== 'running') {
+    console.log("Task must be running for us to connect!");
```

Comment:
I wonder if we could have an option at some point to wait for this artifact to be present so that way we can leave the cli trying to connect and not miss the timeout for an interactive session